pire/extra/count.h

```diff
@@ -203,17 +192,17 @@ class CountingScanner: public LoadedScanner {
 
 	bool Dead(const State&) const { return false; }
 
-	CountingScanner() {}
-	CountingScanner(const CountingScanner& s): LoadedScanner(s) {}
-	CountingScanner(const Fsm& re, const Fsm& sep);
+	BaseCountingScanner() {}
+	BaseCountingScanner(const BaseCountingScanner& s): LoadedScanner(s) {}
 
-	void Swap(CountingScanner& s) { LoadedScanner::Swap(s); }
-	CountingScanner& operator = (const CountingScanner& s) { CountingScanner(s).Swap(*this); return *this; }
+	void Swap(BaseCountingScanner& s) { LoadedScanner::Swap(s); }
+	BaseCountingScanner& operator = (const BaseCountingScanner& s) { BaseCountingScanner(s).Swap(*this); return *this; }
```

Comment:
We could just inherit default constructor, copy constructor, `Swap` and copy assignment operator from LoadedScanner.

Reply:
Done. We need `using LoadedScanner::Swap` to make it public, but all others are inherited implicitly.